sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/PathRemoveAccessControlEntry.java

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.storage.file.datalake.models;
+
+import com.azure.storage.common.implementation.StorageImplUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+import static com.azure.storage.file.datalake.models.PathAccessControlEntry.ACCESS_CONTROL_ENTRY_INVALID_SCOPE;
+import static com.azure.storage.file.datalake.models.PathAccessControlEntry.DEFAULT_SCOPE;
+
+public class PathRemoveAccessControlEntry {
+    private boolean defaultScope;
+    private AccessControlType accessControlType;
+    private String entityId;
+
+    /**
+     * Indicates whether this is the default entry for the ACL.
+     *
+     * @return Whether this is the default entry for the ACL.
+     */
+    public boolean isDefaultScope() {
+        return defaultScope;
+    }
+
+    /**
+     * Sets whether this is the default entry for the ACL.
+     *
+     * @param defaultScope Whether this is the default entry for the ACL.
+     * @return The updated object.
+     */
+    public PathRemoveAccessControlEntry setDefaultScope(boolean defaultScope) {
```

Comment:
Same as with the isDirectory, this probably sounds better as setIsDefaultScope?

Reply:
This one I'd rather keep because it follows the same pattern as PathAccessControlEntry